spec/features/users/user_login_spec.rb

```diff
@@ -40,36 +40,94 @@
         end
       end
     end
+
+    context 'when the user token is expiring' do
+      before do
+        visit_with_expiring_token('/manage_collections')
+      end
+
+      it 'allows access to the manage collections' do
+        expect(page).to have_content('Test User')
+        expect(page).to have_content('Logout')
+      end
+
+      it 'refreshes the user token' do
+        expect(page.get_rack_session_key('access_token')).to eql('new_access_token')
+      end
+    end
   end
 
-  context 'when the user token is expiring' do
+  context 'when logging in with Launchpad' do
     before do
-      real_login
-      visit_with_expiring_token('/manage_collections')
+      require_launchpad_login
+
+      real_login(method: 'launchpad')
     end
 
-    it 'allows access to the manage collections' do
-      expect(page).to have_content('Test User')
-      expect(page).to have_content('Logout')
+    it 'redirects the user to the manage collections page' do
+      within 'h2.current' do
+        expect(page).to have_content('Manage Collections')
+      end
     end
 
-    it 'refreshes the user token' do
-      expect(page.get_rack_session_key('access_token')).to eql('new_access_token')
+    it 'does not display public holdings' do
```

Comment:
is this just to make sure that it isn't still looking at the welcome page? If so, I think it is unnecessary

Reply:
I just copied this from the URS section. I don't think we're even showing it anymore (see comments in `routes.rb` about MMT-867. Should we just remove the tests? Should we take out the route?